qutip/tests/test_lattice.py

```diff
@@ -209,7 +209,7 @@ def test_k(self):
                 else:
                     kop[row, col] = 1 / (np.exp(2j * np.pi * (row - col)/L)-1)
         kt = np.kron(kop * 2 * np.pi / L, np.eye(6))
-        dim_H = [[2, 2, 3], [2, 2, 3]]
+        dim_H = [[L, 2, 3], [L, 2, 3]]
```

Comment:
So this didn't fail before because no operations that checked the dims were used? And it fails now if #1783 is merged?

Reply:
That is my interpretation, yes.